app/services/import_occupation_standard_related_instruction.rb

```diff
@@ -0,0 +1,43 @@
+class ImportOccupationStandardRelatedInstruction
+  attr_reader :occupation_standard, :data_import, :row
+
+  def initialize(occupation_standard:, data_import:)
+    @occupation_standard = occupation_standard
+    @data_import = data_import
+    @row = nil
+  end
+
+  def call
+    related_instructions = []
+    data_import.file.open do |file|
+      xlsx = Roo::Spreadsheet.open(file, extension: :xlsx)
+      sheet = xlsx.sheet(2)
+
+      sheet.parse(headers: true).each_with_index do |row, index|
+        next if index.zero?
+
+        organization = Organization.find_or_create_by!(
+          title: row["Related Training Organization"]
+        )
+
+        course = Course.where(
+          code: row["Course Code"],
+          organization: organization
+        ).first_or_create!(
+          description: row["Course Description"]
+        )
```

Comment:
@jenlu33 I thought that all of these course fields were supposed to actually be on the Related Instruction model? At our meeting before last, I was pretty sure the understanding was that this information wasn't about a specific course in our database but rather general information that would be stored on the Related Instruction record.

Reply:
Some of the fields like code, description, and organization aren't on the Related Instruction model so I wasn't sure where else to store that information.

Reply:
I think we need to add the fields to the RelatedInstruction model.